Form/Type/FormatterType.php

```diff
@@ -129,9 +129,14 @@ public function buildView(FormView $view, FormInterface $form, array $options)
         }
         $view->vars['format_field_options'] = $options['format_field_options'];
 
-        $ckeditorConfiguration = array(
-            'toolbar'       => array_values($options['ckeditor_toolbar_icons']),
-        );
+        $defaultConfig = $this->configManager->getDefaultConfig();
+        if ($this->configManager->hasConfig($defaultConfig)) {
```

Comment:
yeah, you are right, it's not so pretty. I've just copied this code from old pull request..
Fixed.